src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Offence.java

```diff
@@ -87,4 +90,9 @@ public static class PK implements Serializable {
     @BatchSize(size = 25)
     private List<OffenceIndicator> offenceIndicators = new ArrayList<>();
 
+    public boolean isMoreSeriousThan(Offence other) {
+        return isNumeric(this.getSeverityRanking()) && isNumeric(other.getSeverityRanking()) &&
```

Comment:
Why would the severity ranking not be numeric? Just wondering if this could cause problems as `offence1.isMoreSeriousThan(offence2) == false` and `offence2.isMoreSeriousThan(offence1) == false`.

Reply:
So as far as i can see every entry in the dev database is a number. The column is defined as a VARCHAR though. The only way you could have false being returned in both cases as above would be if the column was NULL. I didn't want to change the definition in JPA to be an Integer incase there was something I was missing.